cmd/devtools/run.go

```diff
@@ -11,49 +13,107 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var IsRunLocal bool
+var IsRunRemote bool
+
+type RunConfiguration int
+
+const (
+	Local RunConfiguration = iota
+	Remote
+	Error
```

Comment:
This feels like an anti-pattern. Why not a real error?

Reply:
Fixed with other flag updates.